distribution/helm/vector-aggregator/templates/statefulset.yaml

```diff
@@ -26,10 +26,7 @@ spec:
         {{- include "libvector.selectorLabels" . | nindent 8 }}
         vector.dev/exclude: "true"
     spec:
-      {{- with .Values.imagePullSecrets }}
-      imagePullSecrets:
-        {{- toYaml . | nindent 8 }}
-      {{- end }}
+      {{- include "libvector.imagePullSecrets" . | indent 6 }}
```

Comment:
```suggestion
        {{- include "libvector.imagePullSecrets" . | indent 6 }}
```

Just noticing that it seems like we typically indent the includes under their parent YAML key.

Reply:
This is _actually_ correct, since it's now also rendering the `imagePullSecrets` key, so the parent is the `spec` on line 28

Reply:
Aha, I missed that `imagePullSecrets` was deleted too. :+1: